tests/modules/subcommands.c

```diff
@@ -35,12 +35,26 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     if (RedisModule_Init(ctx, "subcommands", 1, REDISMODULE_APIVER_1) == REDISMODULE_ERR)
         return REDISMODULE_ERR;
 
+    /* Module command names cannot contain special characters. */
+    int count = 0;
+    if (RedisModule_CreateCommand(ctx,"subcommands.char\r",NULL,"",0,0,0) == REDISMODULE_ERR) count++;
+    if (RedisModule_CreateCommand(ctx,"subcommands.char\n",NULL,"",0,0,0) == REDISMODULE_ERR) count++;
+    if (RedisModule_CreateCommand(ctx,"subcommands.char ",NULL,"",0,0,0) == REDISMODULE_ERR) count++;
+    if (count != 3) return REDISMODULE_ERR;
```

Comment:
Is this a common pattern as opposed to something like:
```
RedisModule_Assert(RedisModule_CreateCommand(ctx,"subcommands.char\r",NULL,"",0,0,0) == REDISMODULE_ERR);
```

Reply:
not at all, just the first thing that came into my head back then.
yean, the `RedisModule_Assert` is better, i will change it to this way